Installation/Mapping/oc.githubRepository.mapping.json

```diff
@@ -11,7 +11,12 @@
     "stars": "stargazers_count",
     "fork_count": "forks_count",
     "issue_open_count": "open_issues_count",
-    "source": "github"
+    "source": "github",
+    "forked_from": "{% if fork === true && parent.html_url|default %}parent.html_url{% endif %}",
+    "archived": "{% if archived|default %}{{ archived }}{% endif %}"
+  },
+  "cast": {
+    "archived": "boolean"
```

Comment:
Give me a moment to check, this should have been merged with another PR already.

Reply:
Yup, this was already merged with the previous PR, could you merge from master before merging?